Create-Cluster.md

```diff
@@ -79,11 +97,18 @@ Steps:
           kuberos-b7d5f755d-l8jb5   1/1       Running   0          1m
         ```
 1. Add WebOps group as admins
+
     ```
      $ kubectl apply -f ../../../cluster-config/rbac/webops-cluster-admin.yml
      clusterrolebinding.rbac.authorization.k8s.io "webops-cluster-admin" created
+
     ```
 
+   ### Recovering Environments
+
+1. All the environments created within a cluster is on github, so any trigger of the pipeline will create all environments under namespaces/$cluster_name. You can recover the namespaces and service account via triggering the pipeline.
+
+
 
 
```

Comment:
Recovering deployments 

Go to application repo and kubectl apply the manifest files

Reply:
Switch over DNS